client/components/token-field/index.jsx

```diff
@@ -81,7 +91,8 @@ var TokenField = React.createClass( {
 					scrollIntoView={ this.state.selectedSuggestionScroll }
 					isExpanded={ this.state.isActive }
 					onHover={ this._onSuggestionHovered }
-					onSelect={ this._onSuggestionSelected } />
+					onSelect={ this._onSuggestionSelected }
+				/>
```

Comment:
I'm strongly in favor of putting the `/>` (and even the `>` for tags with children) on the following line, de-indented 1 level:

- Like the `);` and `};` immediately below, it denotes the end of a block
- Easier to rearrange props, like trailing `,` in arrays

We have lots of code in both styles in Calypso.  If we want to do something about this, let's remove that linter rule instead.